pkg/controller/deployment/progress.go

```diff
@@ -188,11 +189,11 @@ func (dc *DeploymentController) requeueStuckDeployment(d *apps.Deployment, newSt
 	// Make it ratelimited so we stay on the safe side, eventually the Deployment should
 	// transition either to a Complete or to a TimedOut condition.
 	if after < time.Second {
-		klog.V(4).Infof("Queueing up deployment %q for a progress check now", d.Name)
+		logger.V(4).Info("Queueing up deployment for a progress check now", "deployment", d.Name)
 		dc.enqueueRateLimited(d)
 		return time.Duration(0)
 	}
-	klog.V(4).Infof("Queueing up deployment %q for a progress check after %ds", d.Name, int(after.Seconds()))
+	logger.V(4).Info("Queueing up deployment for a progress check", "deployment", d.Name, "queueAfter", int(after.Seconds()))
```

Comment:
"deployment" is a Kubernetes object, please log with `klog.KObj`.

Please also check the rest of the PR for this.